packages/network-controller/src/rpc-service/rpc-service.ts

```diff
@@ -468,52 +470,45 @@ export class RpcService implements AbstractRpcService {
    * @throws A generic error if the response HTTP status is not 2xx but also not
    * 405, 429, 503, or 504.
    */
-  async #executePolicy<Result extends Json>(
+  async #processRequest<Result extends Json>(
```

Comment:
Renamed to make it a bit more clear that it's not just executing the policy, it's also processing the response and doing a bunch of error handling.